src/devices/bus/nes/mmc3_clones.cpp

```diff
@@ -950,14 +930,12 @@ void nes_bmc_gn45_device::pcb_reset()
 void nes_bmc_gold7in1_device::device_start()
 {
 	mmc3_start();
-	save_item(NAME(m_reg_written));
+	save_item(NAME(m_lock));
 }
 
 void nes_bmc_gold7in1_device::pcb_reset()
 {
-	m_chr_source = m_vrom_chunks ? CHRROM : CHRRAM;
-
-	m_reg_written = 0;
+	m_lock = 0;
```

Comment:
`m_lock` is a `bool` and you’re assigning it with `bool` literals elsewhere.

Reply:
Old habits die hard. Thanks. Fixed.